requirements.txt

```
@@ -8,8 +8,8 @@ peft>=0.12.0
 decord>=0.6.0
 wandb
 pandas
-torch>=2.4.0
-torchvision>=0.19.0
+torch>=2.5.0
```

I think this will not work at the moment because of the cuDNN errors: https://github.com/huggingface/diffusers/issues/9704

```suggestion
torch<2.5.0
```